templates/course_info.html

```diff
@@ -1,13 +1,9 @@
-{% extends "bootstrap/base.html" %}
-{% block head %}
-  {{ super() }}
-  <link rel="icon" href="{{ url_for('static', filename='favicon.ico') }}" type="image/x-icon">
-{% endblock %}
-{% block title %}{{ subject }}{% endblock %}
+<!-- {% extends "base.html" %}
+{% block title %} Results {% endblock %}
```

Comment:
Make the title of the page more than "Results":
```suggestion
{% block title %} {{ subject|replace('\n', ' ')|safe }} {% endblock %}
```

Reply:
We don't use the course_info.html, we only use the search.html and results.html but I will do that.